src/main/java/ladder/domain/LadderResult.java

```diff
@@ -0,0 +1,26 @@
+package ladder.domain;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LadderResult {
+    private final LinkedHashMap<Member, Reword> rewordMap;
+
+    public LadderResult(LinkedHashMap<Member, Reword> rewordMap) {
+        this.rewordMap = rewordMap;
+    }
+
+
+    public Reword getReword(Member member) {
+        return rewordMap.get(member);
+    }
```

Comment:
```suggestion

    public Reword getReword(Member member) {
        return rewordMap.get(member);
    }
```
이런 부분도 `Member`가 있는지 먼저 검증하면 좋을 것 같아요 :)

Reply:
맞네요! 
수정했습니다~